MIGRATION.md

```diff
@@ -170,6 +170,27 @@ Also make sure you have a `.babelrc` in your project directory. You probably alr
 }
 ```
 
+#### Use with CRA2
```

Comment:
please add a link to this from the table of content at the beginning of the file.
BTW, in v4.1 this is already not needed 😉